paper.tex

```diff
@@ -1360,81 +1360,48 @@ \section*{ARG inference methods}\label{ARG_inference_methods}
+whenever a recombination is traversed. Methods based the backwards-in-time approach are described by
+\citet{song2005efficient, wu2008association, thao2019hybrid, ignatieva2021kwarg},
+while \citet{hein1993heuristic, wu2011new, mirzaei2017rent} describe along-the-genome methods.
+\citet{rasmussen2022espalier} focuses on parsimonious fusion of local trees into an ARG, while
+\citet{camara2016inference} is based on topological data analysis.
 
 Reconstructing a most parsimonious ARG for a given data set is NP-hard \citep{wang2001perfect},
-and in order to allow for larger input datasets, subsequent parsimony-based methods have resorted
-to various relaxations and heuristics. Those utilising variants of the backwards-in-time approach
-include the programs \texttt{SHRUB} \citep{song2005efficient}, \texttt{GAMARG} \citep{thao2019hybrid},
-and \texttt{KwARG} \citep{ignatieva2021kwarg}, and the method described by \citet{wu2008association}.
-Tools constructing genealogies along-the-genome include \texttt{RecPars} \citep{hein1993heuristic},
-\texttt{RENT} \citep{wu2011new} and its sequel \texttt{RENT+} \citep{mirzaei2017rent}. Other approaches
-include \texttt{TARGet} \citep{camara2016inference}, which implements a method based on topological
-data analysis. The scalability of these methods varies substantially based on the heuristics in use
-and the underlying data structure, with the most efficient tools limited to analysing hundreds of
-sequences.
-
-A number of heuristic methods have also been developed which do not (explicitly) focus on reconstructing
-the most parsimonious ARGs, but rather aim for computational efficiency while seeking to achieve good
-accuracy of the inferred genealogies. Such methods include \texttt{Margarita} \citep{minichiello2006mapping}
-and the approach of \citet{parida2008estimating}; the more recently developed \texttt{Relate}
-\citep{speidel2019method} and \texttt{tsinfer} \citep{kelleher2019inferring} use a combination of both
-backwards-in-time and along-the-genome approaches, and can be practically used on megabase scale data
-and tens to hundreds of thousands of samples under human-like parameters.
-Another class of methods has recently emerged that use a \emph{threading} approach, constructing ARGs
-by adding sequences to the graph one-by-one. Originally introduced by \citet{rasmussen2014genome} as a
-way of generating MCMC proposals for ARGs, a similar idea was leveraged in \texttt{ARGneedle}
-\citep{zhang2021biobank} to reconstruct ARGs for biobank-scale data.
+so parsimony-based methods resort to heuristics and are limited to analysing hundreds of sequences.
+Hence, a number of methods aim to balance computational efficiency with reconstruction of ``reasonable",
+rather than parsimonious ARGs
+\citep{minichiello2006mapping, parida2008estimating, kelleher2019inferring,  speidel2019method, zhang2021biobank}.
+The methods of \citet{kelleher2019inferring, speidel2019method, zhang2021biobank, rasmussen2022espalier}
```

Comment:
Perhaps something like "the latter three methods" to avoid repeating these references from the previous sentence?